draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -2202,25 +2418,29 @@ The focus is on client-side protection scheme between IP router and
 reconfigurable ROADM. Scenario here is to define only one port in the
 routers and in the ROADM muxponder board at both ends as back-up
 ports to recover any other port failure on client-side of the ROADM
-(either on the IP router port side or on the muxponder side or on the link
+(either on the IP router port side or on the muxponder side or on the
+link
 between them). When client-side port failure occurs, alarms are
 raised to MDSC by IP-PNC and O-PNC (port status down, LOS etc.). MDSC
 checks with OP-PNC(s) that there is no optical failure in the optical
 layer.
 
 There can be two cases here:
 
-1. LAG was defined between the IP routers at the two ends. MDSC, after checking
-that optical layer is fine between the two edge WDM nodes, triggers
-the WDM edge node re-configuration so that the IP router's back-up port with its
-associated muxponder port can reuse the WDM tunnel that was already in
+1. LAG was defined between the IP routers at the two ends. MDSC, after
+checking
+that optical layer is fine between the two edge DWDM nodes, triggers
+the DWDM edge node re-configuration so that the IP router's back-up port
+with its
+associated muxponder port can reuse the DWDM tunnel that was already in
 use previously by the failed IP router port and adds the new link to
 the LAG on the failure side.
 
    While the ROADM reconfiguration takes place, IP/MPLS traffic is
    using the reduced bandwidth of the IP link bundle, discarding
    lower priority traffic if required. Once back-up port has been
-   reconfigured to reuse the existing WDM tunnel and the new link has been added
+   reconfigured to reuse the existing DWDM tunnel and the new link has
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
   reconfigured to reuse the existing WDM tunnel and the new link has
```